src/lib/verifier/prod.ml

```diff
@@ -230,6 +230,10 @@ module Worker_state = struct
              let set_itn_logger_data ~daemon_port:_ = ()
            end : S )
 
+  let verify_commands (t : t) =
```

Comment:
Also, Verifier_intf seems to already expose this
```ocaml
    val verify_commands :
         t
      -> Mina_base.User_command.Verifiable.t Mina_base.With_status.t list
         (* The first level of error represents failure to verify, the second a failure in
            communicating with the verifier. *)
      -> [ `Valid of Mina_base.User_command.Valid.t
         | `Valid_assuming of
           ( Pickles.Side_loaded.Verification_key.t
           * Mina_base.Zkapp_statement.t
           * Pickles.Side_loaded.Proof.t )
           list
         | invalid ]
         list
         Deferred.Or_error.t
```

May I ask why do you want to expose yet another similar function?

Reply:
That one is implemented by this, if I remember correctly:

https://github.com/MinaProtocol/mina/blob/adb75bc9e3457393b49ed75a258fa7ed6f28b4ab/src/lib/verifier/prod.ml#L647-L654

It spawns a worker to handle the verification request, and requires that there be a full verifier available. I was talking with @georgeee, and he agreed that it was better not to spawn a verifier process just for this benchmark; instead, I exposed the worker state so an individual batch of commands could be tested/benchmarked in isolation, without involving worker spawning or things like that.